### PR TITLE
fix!: Change default from --release to --debug #31

### PR DESCRIPTION
**Pull request by GitaWei**
Hopefully this will resolve the dueling commit issue we had somehow created.  I will use incremental branching for each change moving forward.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -142,9 +142,11 @@ fn parse_args() -> Args {
...
                 .requires("loop")
             )
-            .arg(Arg::new("debug")
-                .help("Build a debug executable, not an optimised one.")
-                .long("debug")
+            .arg(Arg::new("release")
+                .help("Build a release executable, an optimised one.")
+                .short('r')
+                .long("release")
+                .conflicts_with_all(&["bench"])
```
Comment (epage) [inline src/main.rs:149]: How come this is conflicting with `"bench"`?

(sorry, didn't notice this in the prior PR)

Comment (GitaWei) [reply src/main.rs:149]: Maybe I don't fully understand the goals but my decision making was:
1.  Cargo passes the --bench flag to the test harness to tell it to run only benchmarks compile by the bench profile.
2.  The bench profile is the default profile used by cargo bench. While the bench profile inherits the settings from the release profile they may in fact vary and produce a different binary.
3.  We are following the Cargo operational model.

Comment (epage) [reply src/main.rs:149]: You're right; I had gotted mixed with some other things.

I did make `--bench` unstable as a soft-removal, so quality isn't the most important for it.

**Review by GitaWei — COMMENTED** [review]
Appears correct

**Comment by epage** [comment]
> Appears correct

What appears correct?

**Comment by epage** [comment]
> [Merge branch 'main' into rm_debug_#31](https://github.com/epage/cargo-script-mvs/pull/62/commits/34d816ee729c8f99a4727e0b4965f8f935aa2e1b)

FYI I try to avoid merge-commits like this in master.  I can either squash PRs with them or as an alternative, if you rebase against master, rather than merge with master, it'll keep the PRs clean from the start

**Pull request merged**